File: SEproject/Data/Instructor.cs
using System;
using System.Collections.Generic;

namespace SEproject.Data
{
    public partial class Instructor
    {
        public string Id { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string Email { get; set; } = null!;
        public string Position { get; set; } = null!;
        public string Major { get; set; } = null!;
    }
}

File: SEproject/ClientApp/src/app/data/models/instructor.ts
export class Instructor {
    id: string;
    firstName: string;
    lastName: string;
    email: string;
    position: string;
    major: string;

    constructor(id: string, firstName: string, lastName: string, email: string, position: string, major: string) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.position = position;
        this.major = major;
    }
}
